### ISSUE TITLE
Credentials for HTTPS proxy is not updated after changing

### ISSUE THREAD

**Issue by MTProto** [FelisCatus/SwitchyOmega#2084]
### SwitchyOmega version
SwitchyOmega 2.5.21
### Browser version & OS version
Chrome Version 84.0.4147.105 (Official Build) (64-bit)/ Windows 10 64bit
### Problem description
After changing the credentials for an HTTPS proxy, the requests sent to the proxy server, uses the previous credentials, even-though the credentials get updated in the corresponding tab.

Although the new credentials for HTTPS proxy server is changed and saved in the corresponding place in extension, the request sent to proxy server do not use the new credentials, it continues to use the previous one.

#### Steps to reproduce issue / 重现错误所需步骤

(What did you do? / 你做了什么？)

1. Click New Profile, select Proxy Profile and choose a name. Click create, Choose HTTPS as Protocol.
2. Enter a valid HTTPS Proxy Server, enter its Port and click on authentication, Enter a username and a password. click save changes and apply changes. Surf the web.
3. ReEnter to proxy authentication, change either Username or Password or both. Click save changes and then click apply changes.

#### Expected behavior / 期望发生的情况
Extension should use the new credentials to connect to the proxy.

#### Actual (or suggested) behavior / 实际发生的情况（或建议修改后的行为）
Extension, uses the previous credential to connect to proxy, although the credentials is get updated in the corresponding place. But it sends the previous credentials to the proxy server.

**Comment by thedanielius** [comment]
Agree, this is still a serious problem but its browser related. Somehow the credentials are being cached if it uses the same domain(hostname) and there is no way of re-triggering that with new credentials. The only way is to restart the browser..
